WooCommerce/Classes/ViewRelated/Orders/Order Creation/StatusSection/OrderStatusSection.swift

```diff
@@ -60,6 +61,8 @@ private extension OrderStatusSection {
 
     enum Localization {
         static let editButton = NSLocalizedString("Edit", comment: "Button to edit an order status on the New Order screen")
+        static let editButtonAccessibilityLabel = NSLocalizedString("Edit Status",
+                                                                    comment: "Accessibility label for the button to order status on the New Order screen")
```

Comment:
Nit: The comment is missing the word "edit":

```suggestion
                                                                    comment: "Accessibility label for the button to edit order status on the New Order screen")
```